### PR TITLE
fix: replace ticket with case

### PR DESCRIPTION
**Pull request by bradleycamacho**
Hero request

### CODE REVIEW

File: src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx [hunk 1]
```diff
@@ -632,7 +632,7 @@ If you need assistance with New Relic Products, you are in good hands with sever
     id="ticket"
     title="Support case"
   >
-    Support is now available in the New Relic platform! Just click on the question mark at the top right of your New Relic screen to surface contextual documentation and resources. Depending on the Service Level associated with your New Relic account, you may be eligible for ticketed support and can open a ticket without leaving the New Relic Platform. As an alternative, customers eligible for ticketed support may also open a support case from the [New Relic Support page](https://support.newrelic.com/)
+    Support is now available in the New Relic platform! Just click on the question mark at the top right of your New Relic screen to surface contextual documentation and resources. Depending on the Service Level associated with your New Relic account, you may be eligible for cas support and can open a case without leaving the New Relic Platform. As an alternative, customers eligible for case support may also open a support case from the [New Relic Support page](https://support.newrelic.com/)
```
Comment (akristen) [inline src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx:635]:
```suggestion
    Support is now available in the New Relic platform! Just click on the question mark at the top right of your New Relic screen to surface contextual documentation and resources. Depending on the Service Level associated with your New Relic account, you may be eligible for case support and can open a case without leaving the New Relic Platform. As an alternative, customers eligible for case support may also open a support case from the [New Relic Support page](https://support.newrelic.com/)
```

Comment (akristen) [reply src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx:635]: I'll make a quick PR + publish this change myself

File: src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx [hunk 2]
```diff
@@ -736,7 +736,7 @@ We want everyone to monitor their systems, and we're contributing our technology
 
 [New Relic Open Source](https://opensource.newrelic.com/explore-projects/) projects are assigned to one of five different categories. These categories determine the support options available for a project as listed below:
 
-* **Community plus projects**: Actively maintained by New Relic. Support requests can be made through Github, Community, and Ticketed Support channels, depending on the service level associated with the New Relic account.
+* **Community plus projects**: Actively maintained by New Relic. Support requests can be made through Github, Community, and Case Support channels, depending on the service level associated with the New Relic account.
```
Comment (akristen) [inline src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx:739]: Should 'case support channels' be all lowercase? It's more abstract/general than the more specific channels listed in the table.

File: src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx (hunk 1 above)
Comment (akristen) [reply src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx:635]: This isn't actually on the docs site yet so I can't make a quick PR like I said! Will still need a fix

File: src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx (hunk 2 above)
Comment (bradleycamacho) [reply src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx:739]: We tend to let support handle styling of these docs

**Pull request merged**